gtfs_aggregator_checker/cache.py

```diff
@@ -0,0 +1,45 @@
+import os
+from pathlib import Path
+import urllib.error
+import urllib.request
+
+from .utils import url_split
+
+
+def get_cache_dir():
+    if "GTFS_CACHE_DIR" in os.environ:
+        path = Path(os.environ["GTFS_CACHE_DIR"])
+    else:
+        path = Path.home() / ".cache/gtfs-aggregator-checker"
+    path.mkdir(exist_ok=True, parents=True)
+    return path
+
+
+def get_cached(key, func, directory=None):
+    if not directory:
+        directory = get_cache_dir()
+    path = directory / key
+    if not path.exists():
+        content = func()
+        with open(path, "w") as f:
+            f.write(content)
+            print("wrote cached file", path)
```

Comment:
This makes the script pretty noisy the first time it is ran. Can this only be printed out when the `--verbose` option is enabled?